src/Options/docs.js

```diff
@@ -16,7 +16,7 @@
  * @property {Boolean} allowCustomObjectId Enable (or disable) custom objectId
  * @property {Boolean} allowExpiredAuthDataToken Allow a user to log in even if the 3rd party authentication token that was used to sign in to their account has expired. If this is set to `false`, then the token will be validated every time the user signs in to their account. This refers to the token that is stored in the `_User.authData` field. Defaults to `true`.
  * @property {String[]} allowHeaders Add headers to Access-Control-Allow-Headers
- * @property {String} allowOrigin Sets the origin to Access-Control-Allow-Origin
+ * @property {String[]} allowOrigin Sets the origins to Access-Control-Allow-Origin
```

Comment:
This is a breaking change which we cannot merge until Parse Server 7. Could you adapt the PR to accept both, single string and array?

We'd need tests for both and the options validation would need to account for both.

Reply:
Could I change this back to String and add a note indicating that it can be comma delimited to support multiple origins?

Reply:
I've updated the PR to support `string | string[]` for this option

It's currently using the `arrayParser` if a buildConfig has that type, but do you think we should build a new `stringOrStringArrayParser`?

The current `arrayParser` checks for array type or comma separated string

```
function arrayParser(opt) {
  if (Array.isArray(opt)) {
    return opt;
  } else if (typeof opt === 'string') {
    return opt.split(',');
  } else {
    throw new Error(`${opt} should be a comma separated string or an array`);
  }
}
```

but I could build another one that would return the `string` or `string[]` depending on which was used.

Thanks for your time!

Reply:
I think there are some existing options that take a string and array of strings; you could just use that logic.